src/Types.jl

```diff
@@ -336,6 +336,7 @@ Attributes:
 immutable SkyPatch
     center::Vector{Float64}
     radius::Float64
```

Comment:
I don't think the world radius is used anywhere in the code after this change.  Can you delete it?